### ISSUE TITLE
"too many connections" error while seeding from GPKG

### ISSUE THREAD

**Issue by arbakker** [t-rex-tileserver/t-rex#234]
When seeding with a GPKG for a datasource  I see the following error in the log output:

```
ERROR: Too many connections: max 64
```

This got introduced with the Async Seeding (9deb6806ef56cfd34ce7e97c584c61400d571a95) commit. I ran the following command against builds of both 9deb6806ef56cfd34ce7e97c584c61400d571a95 and 762f156 (parent):

```
t_rex generate --config examples/gpkg.toml --loglevel debug &> output-generate-${COMMIT_HASH}.log
```
I noticed that the generate command help stil displays the `nodeno` and `nodes` arguments, is this still working as previously? I figured that maybe the error wont occur when limiting threading to 1 core, but when using the flags  `--nodes 1 --nodeno 0` on 9deb6806ef56cfd34ce7e97c584c61400d571a95  I still see the error `too many connections` appearing. 

Here are the log files and the config file I used:

[gpkg-toml.txt](https://github.com/t-rex-tileserver/t-rex/files/5967018/gpkg-toml.txt)
[output-generate-9deb680.log](https://github.com/t-rex-tileserver/t-rex/files/5967005/output-generate-9deb680.log)
[output-generate-762f156.log](https://github.com/t-rex-tileserver/t-rex/files/5967006/output-generate-762f156.log)

**Comment by pka** [comment]
The `nodeno` and `nodes` arguments are now only meant to distribute tile seeding to multiple servers. Maybe there should be an new option to limit the number of threads used for seeding.

**Comment by pka** [comment]
Should be fixed with e1a67e9
